integration-tests/datasets/arimax_test.csv

```diff
@@ -0,0 +1,11 @@
+SUPPLIER,REF,DATE,VALUE,converusd,petrole,petrole_sin,petrole_log
```

Comment:
Don't use data from Kyle in the tests/examples (since it's from the POC company) - use the air quality or some other data that's open source